src/test/resources/examples/binary/api.yaml

```diff
@@ -0,0 +1,24 @@
+openapi: 3.0.1
```

Comment:
could we tack this example on to an existing test case rather than creating a new example?

Reply:
Sure, which one do you want it added to? From what I can see there are 5 files that currently generate controller code for both spring and micronaut:
- githubApi
- singleAllOf
- authentication
- parameterNameClash
- pathLevelParameters

Based on that I don't think it really fits anywhere?

Reply:
ok, will merge as-is